corehq/toggles.py

```diff
@@ -1735,3 +1735,11 @@ def _commtrackify(domain_name, toggle_is_enabled):
     TAG_INTERNAL,
     namespaces=[NAMESPACE_USER],
 )
+
+
+PARALLEL_MPR_ASR_REPORT = StaticToggle(
+    'parallel_mpr_asr_report',
+    'Release parallel loading of MPR and ASR report',
+    TAG_CUSTOM,
+    [NAMESPACE_DOMAIN]
```

Comment:
Will it be useful to make this accessible to a user so that you can test it out as your user before switching hte domain to it?

(assuming this will be a very short lived toggle)

Reply:
True, I will add